app/src/main/res/layout/notification_dialog.xml

```diff
@@ -53,13 +51,26 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/notificationText" />
 
-    <Button
+    <ImageButton
         android:id="@+id/closeButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="16dp"
         android:layout_marginTop="4dp"
-        android:text="Close"
+        android:layout_marginEnd="16dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@android:drawable/ic_menu_close_clear_cancel" />
```

Comment:
Your previous button to close out of the dialog box felt out of flow on the user interface. Maybe it was because it was a text button, and typically exit buttons are displayed with a symbol. Here I replaced the text button with an image button, where the image is an exit symbol. I also moved the exit button to the top right of the ui, where I think it feels more natural.

Reply:
Pictures are faster than words.  Great idea.